api/aboutpage/models/aboutpage.settings.json

```diff
@@ -50,9 +52,7 @@
       "type": "text"
     },
     "button": {
-      "type": "string"
+      "type": "text"
```

Comment:
Okay, shouldn't the button be `string`? Or, do we really expect a paragraph-long button? :-P